lib/widgets/close_popup.dart

```diff
@@ -13,7 +12,7 @@ WillPopCallback willPopCallback(
     if (canCancel != null && canCancel()) return true;
     return promptAreYouSure(context, title, Text(message)).then((shouldExit) {
       if (shouldExit) {
-        exit(0);
+        SystemNavigator.pop(animated: true);
```

Comment:
Shouldn't we just return true here for the default exit behaviour?

Reply:
You nailed it 😄